uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestTest.php

```diff
@@ -30,11 +31,14 @@ class CRM_HRLeaveAndAbsences_Service_LeaveRequestTest extends BaseHeadlessTest {
 
   private $leaveContact;
 
+  private $absenceType;
+
   public function setUp() {
     CRM_Core_DAO::executeQuery("SET foreign_key_checks = 0;");
     $this->leaveBalanceChangeService = new LeaveBalanceChangeService();
 
     $this->leaveContact = 1;
+    $this->absenceType = AbsenceTypeFabricator::fabricate();
```

Comment:
I thought creating it in setup would just create it once and it can be used in other tests whereas if I create it in each of the seven tests it will create it seven times. Is that correct?

Reply:
Not really. The setUp method is called once before each test, which means it will be called 30 times.